src/config.c

```diff
@@ -628,6 +627,7 @@ void loadServerConfigFromString(char *config) {
     return;
 
 loaderr:
+    if (argv) sdsfreesplitres(argv,argc);
```

Comment:
now you need to make sure to nullify it after the other calls to sdsfreesplitres, otherwise we risk double free.
for example, there's a case were we `goto loaderr` before allocating it (the `Unbalanced quotes in configuration line`), so if we released it on a previous loop, and then in the next loop we fall into that error, we'll have a double free and crash.

Reply:
Updated argv with null where the free is being called other than loaderr, hopefully this covers all the cases and also updated the for new_argv created part of one of the if check.